### DISCUSSION TITLE
Is the documentation of Nuclei updated?

### DISCUSSION THREAD

**Discussion by fmateosb** [projectdiscovery/nuclei#2162] [General]
Hi, I found new features like -resume, -automatic-scan and others and I cannot see them in the wikipage https://nuclei.projectdiscovery.io/nuclei/get-started/

Are there any wiki or documentation explaining how to use nuclei with the last release?

Thanks.

**Comment by forgedhallpass** [comment]
Hello @fmateosb,

Thank you for pointing this out.

* `-resume` flag as the name suggests, enables you to resume a scan that was interrupted by a SIGINT (CTRL + C)
* `-automatic-scan` uses `wappalyzer` to detect the different technologies used by any given hosts, and the findings are mapped to existing template tags to be executed. In short, it is an optimization that only executes templates that are considered relevant for a specific target.

That being said, I've also created a [ticket](https://github.com/projectdiscovery/nuclei-docs/issues/44) for you on the [nuclei-docs](https://github.com/projectdiscovery/nuclei-docs) repository, where the nuclei documentation lives.
